dhash.py

```diff
@@ -14,8 +14,8 @@
 
 # Allow library to be imported even if neither wand or PIL are installed
 try:
-    import wand.image
     import wand.color
+    from wand.image import Image as WandImage
```

Comment:
Per above, please revert this stylistic change.

Reply:
reverted!